### PR TITLE
chore: added github actions

### PR DESCRIPTION
**Pull request by matveypashkovskiy**

### CODE REVIEW

**Comment by matveypashkovskiy** [comment]
we either have to configure black or fix all the issues

**Comment by guotin** [comment]
Currently working on fixing all the issues with `tox`

**Comment by matveypashkovskiy** [comment]
sorry for span, looks like I marked you as reviewers too early

File: tox.ini [hunk 1]
```diff
@@ -1,19 +1,18 @@
 [tox]
 requires = tox-venv
 skipsdist = True
-envlist = py36,py37,py38,py39,py38-check-format
+envlist = py36,py37,py38,py39
 
 [testenv]
 deps = .[dev]
 commands =
+        black --check tests_selector --exclude helper_project
```
Comment (alexey-vyskubov) [inline tox.ini:9]: Different versions of Python might (theoretically) have different versions of black; also there's no need to check formatting 4 times. That's why I had it in a separate testenv.

**Review by alexey-vyskubov — CHANGES_REQUESTED** [review]
I don't like the idea of checking the formatting 4 times.

File: tox.ini (hunk 1 above)
Comment (matveypashkovskiy) [reply tox.ini:9]: it is clear but, I wasn't sure how to configure it with github actions, can you give me a hand her?

Comment (alexey-vyskubov) [reply tox.ini:9]: Just omit black call from `[testenv]` and add the following in the end of the `tox.ini`:
```
# No need to run black with every possible Python version
[testenv:py38]
commands_pre = black --check tests_selector --exclude helper_project
```

Now for py38 environment black will be run before anything else.

Comment (matveypashkovskiy) [reply tox.ini:9]: done

**Review by alexey-vyskubov — APPROVED** [review]
Go-go-go!

**Pull request merged**